pkg/image/context.go

```diff
@@ -17,7 +17,7 @@ type kubernetesScriptInstaller interface {
 }
 
 type kubernetesArtefactDownloader interface {
-	DownloadArtefacts(kubernetes Kubernetes, arch Arch, destinationPath string) error
+	DownloadArtefacts(kubernetes Kubernetes, arch Arch, destinationPath string) (string, string, error)
```

Comment:
We talked about this in chat. Noting it here that they aren't named in the interface but are in the implementation. Your call if you want to change it.